fhir-swagger-generator/src/main/java/com/ibm/fhir/openapi/generator/FHIROpenApiGenerator.java

```diff
@@ -1465,10 +1477,6 @@ private static void generateWholeSystemHistoryPathItem(JsonObjectBuilder path, S
         }
         get.add("operationId", operationId);
 
-        JsonArrayBuilder produces = factory.createArrayBuilder();
-        produces.add(FHIRMediaType.APPLICATION_FHIR_JSON);
-        get.add("produces", produces);
```

Comment:
I'd expect this to exist going forward. Where is it set if it's not compliant?

Reply:
Top level produces array has been removed.  It is specified in the responses content.  For example:
                "responses": {
                    "200": {
                        "description": "Search Condition operation successful",
                        "content": {
                            "application/fhir+json": {
                                "schema": {
                                    "$ref": "#/components/schemas/Bundle"
                                }
                            }
                        }
                    }
                }

Reply:
Right, not what I would expect, since you can specify Content-Type as one (say application/fhir+json), and Accept as another (say application/fhir+xml).  Just a bit surprising.

Reply:
Accept type would be specified in the requestBody.  Similar pattern:

```
                "requestBody": {
                    "content": {
                        "application/x-www-form-urlencoded": {
                            "schema": {
                                "type": "object",
                                "properties": {
                                }
                            }
                        }
                    }
                },

```